CBOnlineIos/countryData.json

```diff
@@ -0,0 +1,3912 @@
+[
+
+    {
+        "CountryID": 93,
```

Comment:
We don't want this thing to be managed by ourselves so i would suggest using https://github.com/chronotruck/FlagPhoneNumber

Reply:
kindly remove any unused file/code